Dev/Cpp/Viewer/GUI/efk.GUIManager.cpp

```diff
@@ -1109,7 +1109,7 @@ namespace efk
 		return Vec2(v.x, v.y);
 	}
 
-	void GUIManager::SetNextWindowPos(const Vec2& pos, Cond cond, const const Vec2& pivot)
```

Comment:
I think this particular change is not correct, tell me how I can escape the `duplicate ‘const’` error here ?

Reply:
It is a my mistake. Please remove first const.